libs/domain/checkout/customer/customer.component.spec.ts

```diff
@@ -108,15 +108,21 @@ describe('CheckoutAuthComponent', () => {
   describe('when the user is authenticated', () => {
     beforeEach(async () => {
       authService.isAuthenticated.mockReturnValue(of(true));
-      userService.getUser.mockReturnValue(of({ email: 'foo@bar.com' } as User));
+      userService.getUser.mockReturnValue(
```

Comment:
I do not see any reason to check heading's content, it is not conditional. I would suggest checking it this way:

```
const user = { firstName: 'Foo', lastName: 'Bar', email: 'foo@bar.com' } as User;
 beforeEach(async () => {
      authService.isAuthenticated.mockReturnValue(of(true));
      userService.getUser.mockReturnValue(
        of(user)
      );
      element = await fixture(
        html`<oryx-checkout-customer></oryx-checkout-customer>`
      );
    });

    it('should render user`s data', () => {
      const customerData = element.renderRoot.querySelector('p')?.textContent;
      expect(customerData).toContain(user.firstName);
      expect(customerData).toContain(user.lastName);
      expect(customerData).toContain(user.email);
    });
```